frontend/app/work_packages/directives/inplace_editor/inplace-editor-display-pane-directive.js

```diff
@@ -92,10 +92,13 @@ module.exports = function(
         scope.displayPaneController.placeholder = I18n.t('js.label_click_to_enter_description');
       }
 
-      scope.$watch('fieldController.isEditing', function(isEditing) {
+      scope.$watch('fieldController.isEditing', function(isEditing, oldIsEditing) {
         if (!isEditing) {
           $timeout(function() {
-            element.find('.inplace-editing--trigger-link').focus();
+            if (oldIsEditing) {
```

Comment:
1. isEditing is true for when the editing view is visible and false for when the display part is visible.
2. the watcher are triggered on linking no matter what, probably because undefined | null -> false, in which case the old isEditing value would be falsy anyway, meaning that it's the very first watch triggered and i don't need to focus crayon.